src/debug/debug_stream/debug_stream_slot.c

```diff
@@ -47,7 +46,7 @@ debug_stream_get_circular_buffer(struct debug_stream_section_descriptor *desc, u
 
 int debug_stream_slot_send_record(struct debug_stream_record *rec)
 {
-	struct debug_stream_section_descriptor desc;
+	struct debug_stream_section_descriptor desc = { 0 };
```

Comment:
weird, do you really see the compiler complain about this? You have a very similar case below at line 136 (137) and there it doesn't complain? From my highly incomplete and imprecise statistics, I wouldn't expect any compiler complaints here

Reply:
yup, either fix all or don't fix any

Reply:
The compiler does not complain, because 'desc' variable is never used there, so the compiler has no reason to worry if its uninitialized or not. But sure, I can initialize it there too.